hyperjumptech/monika
# Added: isEmpty alert query helper

# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1. Resolves #428 

## How did you implement / how did you fix it  
1.  Added `isEmpty` helper for alert query

## How to test  
1. Run monika with this config

```yml
notifications:
  - id: desktop
    type: desktop
probes:
  - id: sample_1
    name: Sample Request
    requests:
      - method: GET
        url: https://reqres.in/api/users/23
    alerts:
      - query: response.time > 200
        message: Response time is {{ response.time }} ms, expecting less than 200ms
      - query: response.status != 200
        message: Status code is not 200. Did you fix something in production?!
      - query: isEmpty(response.body)
        message: Hmm, I expect user data but it is empty.
    incidentThreshold: 2
    recoveryThreshold: 2

```
